hugolib/page.go

```diff
@@ -330,13 +330,69 @@ func (p *Page) Param(key interface{}) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	keyStr = strings.ToLower(keyStr)
+	result, _ := p.traverseDirect(keyStr)
+	if result != nil {
+		return result, nil
+	}
+
+	keySegments := strings.Split(keyStr, ".")
+	if len(keySegments) == 1 {
+		return nil, nil
+	} else {
+		result, _ = p.traverseNested(keySegments)
+		if result != nil {
+			return result, nil
+		}
+	}
+
+	return nil, nil
+}
```

Comment:
Don't ignore the error from `traverseNested()`.  The end of this func could be simplified to:

```go
if len(keySegments) == 1 {
    return nil, nil
}

return p.traverseNested(keySegments)
```